core-plugins/src/test/java/co/cask/hydrator/plugin/batch/aggregator/DistinctTestRun.java

```diff
@@ -60,12 +61,12 @@ public void testDistinct() throws Exception {
     Schema inputSchema = Schema.recordOf(
       "purchase",
       Schema.Field.of("ts", Schema.of(Schema.Type.LONG)),
-      Schema.Field.of("user", Schema.of(Schema.Type.STRING)),
+      Schema.Field.of("user_name", Schema.of(Schema.Type.STRING)),
```

Comment:
added this change since table does not get created in hive because of `user` keyword.